.github/workflows/ci-test.yml

```diff
@@ -39,7 +40,26 @@ jobs:
            </servers>
           </settings>
           EOF
-      - name: Build with Maven
+      - name: Clone CCM
+        # 24-Jun-2022, tatu: CCM install copied from C2 repo's ".github/workflows/ci.yml":
+        uses: actions/checkout@v3
+        with:
+          repository: riptano/ccm
+          ref: master
+          fetch-depth: 1
+          path: ccm
+      - name: Install CCM
+        run: |
+          sudo apt-get install python2.7
+          sudo apt-get install python-dev
+          sudo apt-get install python-setuptools
+          sudo apt-get install python-yaml
+          python -m pip install --upgrade pip
+          python -m pip install psutil
+          cd ccm
+          sudo ./setup.py install
+          ccm list
+      - name: Run Integration Tests against C*4.0
         run: |
-          ./mvnw verify -P it-cassandra-4.0
+          ./mvnw verify -B -ntp -P it-cassandra-4.0
```

Comment:
`-ntp` stands for "no transfer progress" to avoid logging progress bar updates on log output (only makes sense on console). `-B` is for Batch processing, typically used for CI.

To further reduce logging, could consider `-q` for "quiet" but leaving that out for now.